dyno-demo/src/main/java/com/netflix/dyno/demo/redis/DynoJedisDemo.java

```diff
@@ -983,7 +1001,7 @@ public static void main(String args[]) throws IOException {
 			}
 			case 5: {
 				final boolean writeKeys = Boolean.valueOf(props.getProperty("dyno.demo.scan.populateKeys"));
```

Comment:
Apparently, `writeKeys` is read by the property file but not used as `demo.runScanTest(true)`, always defaults to `true`.

Reply:
oh sorry about that.